packages/dapp/src/actionCreators/challenges.ts

```diff
@@ -19,3 +24,31 @@ export const addUserChallengeData = (challengeID: string, user: EthAddress, user
     data: { challengeID, user, userChallengeData },
   };
 };
+
+export const fetchChallenge = (data: any): AnyAction => {
```

Comment:
I wonder if this action would make more sense split up into 2, 1 for "begin fetch", 1 for "end fetch" with the only parameter going into the action is the challenge ID